shop/migrations/0001_initial.py

```diff
@@ -0,0 +1,160 @@
+            ],
+            options={
+                'ordering': ('user',),
+            },
+        ),
+        migrations.CreateModel(
+            name='Product',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(db_index=True, max_length=200)),
+                ('price', models.DecimalField(decimal_places=2, max_digits=10)),
+                ('description', models.TextField()),
+                ('materials', models.CharField(max_length=150)),
+                ('size', models.CharField(max_length=10)),
+                ('weight', models.FloatField()),
+                ('stock', models.PositiveSmallIntegerField()),
+                ('is_available', models.BooleanField(default=True)),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('category', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='products', to='shop.category')),
+            ],
+            options={
+                'ordering': ('category', 'name'),
+            },
+        ),
+        migrations.CreateModel(
+            name='OrderItem',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('quantity', models.PositiveSmallIntegerField()),
+                ('order', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='order_items', to='shop.order')),
+                ('product', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='order_items', to='shop.product')),
+            ],
+            options={
+                'ordering': ('order', 'product'),
+            },
+        ),
+        migrations.CreateModel(
+            name='Image',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('image', models.ImageField(upload_to='')),
+                ('tip', models.CharField(max_length=100)),
+                ('feedback', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='images', to='shop.feedback')),
+                ('product', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='images', to='shop.product')),
+            ],
+            options={
+                'ordering': ('product', 'feedback'),
+            },
+        ),
+        migrations.AddField(
+            model_name='feedback',
+            name='product',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='feedback', to='shop.product'),
```

Comment:
what does it mean?

Reply:
The field of Feedback model. I don't know why it is placed here. Maybe I added this field later.

Reply:
![image](https://user-images.githubusercontent.com/86672602/124748380-b77de100-df2b-11eb-968b-09adbfb66038.png)
Here is the output of the `python manage.py makemigrations`. As I can see, this field was added at the end because the `Product` model was created after the creation of the `Feedback` model. Thus, the `product` field of the `Feedback` model couldn't be created when creating the whole `Feedback`.